bot/cogs/verification.py

```diff
@@ -32,26 +59,409 @@
 <#{constants.Channels.bot_commands}>.
 """
 
-BOT_MESSAGE_DELETE_DELAY = 10
+# Sent via DMs to users kicked for failing to verify
+KICKED_MESSAGE = f"""
+Hi! You have been automatically kicked from Python Discord as you have failed to accept our rules \
+within `{KICKED_AFTER}` days. If this was an accident, please feel free to join us again!
+{constants.Guild.invite}
+"""
```

Comment:
I think an extra newline may look tidier.

```suggestion
KICKED_MESSAGE = f"""
Hi! You have been automatically kicked from Python Discord as you have failed to accept our rules \
within `{KICKED_AFTER}` days. If this was an accident, please feel free to join us again!

{constants.Guild.invite}
"""
```

Reply:
Newlinered in 3ea04e3.

![john_newlineman](https://user-images.githubusercontent.com/44734341/91333149-80985400-e7cd-11ea-959e-5f783302c94b.png)